### DISCUSSION TITLE
Any recommended update form (pre-populated) tutorial?

### DISCUSSION THREAD

**Discussion by Cuantico** [react-hook-form/react-hook-form#5156] [Q&A]
Any recommended update form (pre-populated) tutorial?

**Comment by bluebill1049** [comment]
what do you mean pre populated?

**Reply by Cuantico** [reply]
Thanks for answer, basically by pre populated I mean a form that is build for editing or to modify the loaded values, not a new for that is populated typing in the fields.

**Reply by bluebill1049** [reply]
I think you can use reset at useEffect.

**Reply by Cuantico** [reply]
I think that update forms are not supported.
